distribution/ddf-common/src/main/resources/bin/karaf

```diff
@@ -0,0 +1,331 @@
+#!/bin/sh
```

Comment:
❓ Is it possible to add comments as to what needed to be changed in this file? This would help the next team that needs to perform a karaf upgrade.

Reply:
They had a bug in their file, it looks like they fixed the ticket I wrote.